Library/Formula/wdt.rb

```diff
@@ -0,0 +1,90 @@
+class Wdt < Formula
+  desc "Transfer data as fast as possible over multiple TCP paths."
+  homepage "https://github.com/facebook/wdt"
+
+  stable do
+    url "https://github.com/facebook/wdt/archive/v1.26.1603040.tar.gz"
+    sha256 "b0fffdd12478ef74de50852b9c356352dae5f7ccee22dbca91a88afc6c5b6fee"
+
+    resource "folly" do
+      url "https://github.com/facebook/folly/archive/v0.57.0.tar.gz"
+      sha256 "92fc421e5ea4283e3c515d6062cb1b7ef21965621544f4f85a2251455e034e4b"
+    end
+
+    patch :DATA
+  end
+
+  head do
+    url "https://github.com/facebook/wdt.git"
+
+    resource "folly" do
+      url "https://github.com/facebook/folly.git"
+    end
+  end
+
+  depends_on "cmake" => :build
+  depends_on "gflags" => :build
+  depends_on "glog" => :build
+  depends_on "double-conversion" => :build
+  depends_on "openssl"
+  depends_on "folly"
```

Comment:
We use resourced version of folly, because wdt use some .cpp from folly, on the other hand,  we still need depends_on folly, because wdt header files including folly.